src/thunderbots/software/backend_output/main.cpp

```diff
@@ -5,11 +5,14 @@
 
 #include <iostream>
 
+#include "ai/ai.h"
```

Comment:
What do we need `ai.h` for here?

Reply:
I'm confused. The `AI` class doesn't use the `backend_output`, and even if it did, the `AI` class would include the `backend_output`. I see no reason for this file to include `ai.h`. Try remove it, see if it compiles (it should), and commit that.

Reply:
Oh woops, I checked the wrong file. I think you're right.

Reply:
removed.